### PR TITLE
View binding support added

### PR DESCRIPTION
**Pull request by atetc**
Support of ViewBinding added using official [documentation ](https://developer.android.com/topic/libraries/view-binding) and [sample](https://github.com/android/architecture-components-samples/tree/master/ViewBindingSample)

### CODE REVIEW

File: application/buildSrc/src/main/java/com/stepango/forma/feature/AndroidLibrary.kt [hunk 1]
```diff
@@ -15,7 +15,8 @@ data class AndroidLibraryFeatureConfiguration(
     val consumerMinificationFiles: Set<String> = emptySet(),
     val manifestPlaceholders: Map<String, Any> = emptyMap(),
     val selfValidator: Validator = validator(LibraryModule),
-    val dataBinding: Boolean = false
+    val dataBinding: Boolean = false,
```
Comment (stepango) [inline application/buildSrc/src/main/java/com/stepango/forma/feature/AndroidLibrary.kt:18]: Hi, @atetc nice to see you here! 
dataBindings removed from androidLibrary implementation it is a different module type now, viewBinding should not be allowed in androidLibrary as well. please reference dataBindings module type implementation for more details

**Comment by atetc** [comment]
@stepango you could add WIP (work in progress) label too

**Comment by stepango** [comment]
Converted to draft

**Comment by atetc** [comment]
Seems to me view binding is working on this PR and on my project (I used this repo as a template for my project)
You could consider to merge this PR and apply changes by you/project vision roadmap later.
I am going to stop with this PR for now.

**Comment by stepango** [comment]
Feel free to reopen once you decide to continue working on it 👍

**Pull request closed without merging**